packages/core/src/components/modal/modal.scss

```diff
@@ -92,20 +92,12 @@ $modals: (
 
 @mixin modal-header {
   display: flex;
-  padding-bottom: var(--tds-spacing-element-8);
+  padding: var(--tds-spacing-element-8);
+  padding: 16px;
```

Comment:
Do we need both padding declarations? I feel like they might just override each other, but maybe I'm missing something? :)